docker/primer_schemes/ARTIC/SARS-CoV-2/V4/SARS-CoV-2.scheme.fasta

```diff
@@ -155,7 +155,7 @@ CAAACCTTTTGAGAGAGATATTTCAACTGA
 CACTGACACCACCAAAAGAACATG
 CTGAGTCTAACAAAAAGTTTCTGCCTT
 GGATTGACTAGCTACACTACGTGC
-ACCCATTGGTGCAGGTATAGC
+ACCCATTGGTGCAGGTATATGC
```

Comment:
Interestingly, there is a base which mismatches.
Considering that anything else matches, I would be tempted to think that the original sequence was incorrect.